core/blockchain_test.go

```diff
@@ -2519,7 +2519,7 @@ func testInsertKnownChainDataWithMerging(t *testing.T, typ string, mergeHeight i
 			}
 		}
 	} else {
-		inserter = func(blocks []*types.Block, receipts []types.Receipts) error {
+		inserter = func(blocks []*types.Block, _ []types.Receipts) error {
```

Comment:
same here

Reply:
yes, it is a bit unnecessary. And test ci has already checked this functions. I have restored this functions.